### PR TITLE
Mpvanderschelling/issue189

### PR DESCRIPTION
**Pull request by mpvanderschelling**

### CODE REVIEW

File: src/f3dasm/_src/experimentdata/experimentdata.py [hunk 1]
```diff
@@ -171,6 +177,20 @@ def wrapper_func(self, *args, **kwargs) -> None:
...
+
+    @property
+    def index(self) -> pd.Index:
+        """Returns the index of the ExperimentData
+
+        Returns
+        -------
+        pd.Index
+            The indices of the experiments
+        """
```
Comment (BernardoFerreira) [inline src/f3dasm/_src/experimentdata/experimentdata.py:191]: Method and return docstrings don't seem consistent with each other and are not clear.

File: src/f3dasm/_src/experimentdata/experimentdata.py [hunk 2]
```diff
@@ -486,7 +506,7 @@ def _reset_index(self) -> None:
     #                                                                        ExperimentSample
```
Comment (BernardoFerreira) [inline src/f3dasm/_src/experimentdata/experimentdata.py:506]: Alignment typo.

**Review by BernardoFerreira — CHANGES_REQUESTED** [review]
Expected method to extract sample by index.

Please check the review comments on the code.

I suggest enriching this method to extract multiple samples (e.g., tuple[ExperimentSample]) from an iterable of indexes **or** create a new method (e.g., get_experiment_samples(...)) for this purpose. This is a simple improvement that avoids burdening the user with the task of implementing an explicit cycle to extract multiple samples.

File: src/f3dasm/_src/experimentdata/experimentdata.py (hunk 1 above)
Comment (mpvanderschelling) [reply src/f3dasm/_src/experimentdata/experimentdata.py:191]: I made the description of this docstring more descriptive and added clarity to the return type

File: src/f3dasm/_src/experimentdata/experimentdata.py (hunk 2 above)
Comment (mpvanderschelling) [reply src/f3dasm/_src/experimentdata/experimentdata.py:506]: I unindented the line!

**Comment by mpvanderschelling** [comment]
> Expected method to extract sample by index.
> 
> Please check the review comments on the code.
> 
> I suggest enriching this method to extract multiple samples (e.g., tuple[ExperimentSample]) from an iterable of indexes **or** create a new method (e.g., get_experiment_samples(...)) for this purpose. This is a simple improvement that avoids burdening the user with the task of implementing an explicit cycle to extract multiple samples.

I have created a new issue #205 to address this feature

**Review by BernardoFerreira — APPROVED** [review]

**Pull request merged**